cpp/ql/src/semmle/code/cpp/exprs/Call.qll

```diff
@@ -141,15 +141,27 @@ class FunctionCall extends Call, @funbindexpr {
   /** Gets an explicit template argument for this call. */
   Type getAnExplicitTemplateArgument() { result = getExplicitTemplateArgument(_) }
 
+  /** Gets an explicit template argument value for this call. */
+  Expr getAnExplicitTemplateArgumentValue() { result = getExplicitTemplateArgumentValue(_) }
+
   /** Gets a template argument for this call. */
   Type getATemplateArgument() { result = getTarget().getATemplateArgument() }
 
+  /** Gets a template argument value for this call. */
+  Expr getATemplateArgumentValue() { result = getTarget().getATemplateArgumentValue() }
+
   /** Gets the nth explicit template argument for this call. */
   Type getExplicitTemplateArgument(int n) {
     n < getNumberOfExplicitTemplateArguments() and
     result = getTemplateArgument(n)
   }
 
+  /** Gets the nth explicit template argument value for this call. */
+  Expr getExplicitTemplateArgumentValue(int n) {
+    n < getNumberOfExplicitTemplateArguments() and
```

Comment:
Is this line needed?

Reply:
I don't know - I copied it from getExplicitTemplateArgument() above.

Reply:
Okay, no worries.